tests/standalone/run_all.py

```diff
@@ -281,125 +290,79 @@
 
     extra_flags = ["expect_success", "standalone", "remove_output"]
 
-    if filename == "PySideUsing.py":
+    # specials: these modules need custom checks(implemented below), skip automated checking
+    specials = ['TkInterUsing.py']
+    # skip each test if their respective required import is not present
+    required_m = find_required_module(filename)
```

Comment:
Please name it `required_module`, do not abbreviate things that hurt readability.